countersheet.py

```diff
@@ -1402,7 +1402,7 @@ def effect(self):
                                    docwidth,
                                    self.getViewBoxHeight(svg))]
             margin = max(self.registrationmarkslen,
-                         self.outlinedist)
+                         self.outlinedist) + self.registrationmarksdist
```

Comment:
I think it should only be added to registrationmarkslen (in the call to max), not added like that?

Reply:
Done. :-)